flexibee-core/pom.xml

```diff
@@ -10,6 +10,11 @@
     <artifactId>flexibee-core</artifactId>
 
     <dependencies>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-web</artifactId>
```

Comment:
Neni mi jasny co tady presne dela Spring Boot, chtelo by to vysvetlit. flexibee-core je knihovna, ktera nema mit duvod pro hard dependency na Spring Boot web starter. Takovou dependency maji mit sluzby, nikoliv knihovny.